frontends/web/src/utils/config.ts

```diff
@@ -25,12 +25,21 @@ type TConfig = {
 let pendingConfig: TConfig = {};
 
 /**
- * expects an object with a backend or frontend key
- * i.e. { frontend: { language }}
+ * get current configs
+ * i.e. await getConfig()
+ * returns a promise with backend and frontend configs
+ */
+export const getConfig = (): Promise<any> => {
+  return apiGet('config');
+};
```

Comment:
Ideally we wouldn't use any for config, I tired with unknown.. something like following, but ran into too many issues. 

```diff
diff --git a/frontends/web/src/utils/config.ts b/frontends/web/src/utils/config.ts
index 581325412..f4cbae175 100644
--- a/frontends/web/src/utils/config.ts
+++ b/frontends/web/src/utils/config.ts
@@ -18,18 +18,22 @@
 import { apiGet, apiPost } from '../utils/request';
 
 type TConfig = {
-  backend?: unknown;
-  frontend?: unknown;
+  backend: {
+    [key: string]: unknown
+  };
+  frontend: {
+    [key: string]: unknown
+  };
 };
 
-let pendingConfig: TConfig = {};
+let pendingConfig: Partial<TConfig> = {};
 
 /**
  * get current configs
  * i.e. await getConfig()
  * returns a promise with backend and frontend configs
  */
-export const getConfig = (): Promise<any> => {
+export const getConfig = (): Promise<TConfig> => {
   return apiGet('config');
 };
 
@@ -38,9 +42,9 @@ export const getConfig = (): Promise<any> => {
  * i.e. await setConfig({ frontend: { language }})
  * returns a promise and passes the new config
  */
-export const setConfig = (object: TConfig) => {
+export const setConfig = (object: Partial<TConfig>) => {
   return getConfig()
-    .then((currentConfig = {}) => {
+    .then((currentConfig) => {
       const nextConfig = Object.assign(currentConfig, {
         backend: Object.assign({}, currentConfig.backend, pendingConfig.backend, object.backend),
         frontend: Object.assign({}, currentConfig.frontend, pendingConfig.frontend, object.frontend)
```